<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="app can be shared to another users landing page" timestamp="2016-01-29T15:48:06" hostname="localhost" time="94.063" errors="0" tests="1" skipped="0" disabled="0" failures="2">
  <testcase classname="app can be shared to another users landing page" name="1. publish app to another user" time="60.166">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at [object Object]._onTimeout (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1782:23)
    at Timer.listOnTimeout (timers.js:92:15)]]>
   </failure>
   <failure type="exception" message="Failed: No element found using locator: By.xpath(&quot;.//*[@id=&apos;kickstart&apos;]//a&quot;)"><![CDATA[Error: Failed: No element found using locator: By.xpath(".//*[@id='kickstart']//a")
    at stack (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1441:17)
    at buildExpectationResult (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1411:14)
    at Spec.Env.expectationResultFactory (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:533:18)
    at Spec.addExpectationResult (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:293:34)
    at Env.fail (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:837:25)
    at Function.next.fail (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1776:19)
    at /Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/jasminewd2/index.js:104:16
    at /Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/lib/goog/base.js:1582:15
    at [object Object].webdriver.promise.ControlFlow.runInNewFrame_ (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver/promise.js:1654:20)
    at notify (/Users/hashraf/activiti-qa/protractor-tests/node_modules/grunt-protractor-runner/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver/promise.js:465:12)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>